.circleci/config.yml

```diff
@@ -63,8 +63,84 @@ jobs:
       - BST_CONTAINER: ec2-user@ec2-18-232-68-210.compute-1.amazonaws.com
     working_directory: ~/circleci-deployment
     steps: *deploy-steps
+  
+  skill-testing-release:
+    docker:
+      - image: circleci/node:8
+    environment:
+      - ASK_ACCESS_TOKEN: dummy
+      - ASK_REFRESH_TOKEN: dummy
+      - ASK_VENDOR_ID: dummy
+      - ASK_SKILL_ID: dummy
+      - VIRTUAL_DEVICE_TOKEN: dummy
+    working_directory: ~/skill-testing-release
+    steps:
+      - add_ssh_keys:
+          fingerprints:
+            - "cf:77:b3:3a:08:72:b8:ad:bf:c4:11:97:3b:18:85:76"
+      - checkout
+      - run: git config --global user.email "$GITHUB_USER_EMAIL_RELEASE"
+      - run: git config --global user.name "$GITHUB_USER_NAME_RELEASE"
+      - run: echo "//registry.npmjs.org/:_authToken=$NPM_AUTH_TOKEN" > ~/.npmrc
+      - run: git clone git@github.com:bespoken/skill-testing-ml.git
+      - run: cd skill-testing-ml && npm install
+      - run: cd skill-testing-ml && npm --no-git-tag-version version patch
+      - run:
+          name: get version
+          command: |
+            cd skill-testing-ml
+            SKILL_TESTING_VERSION=$(node -pe "require('./package.json').version")
+            echo "export VERSION=$SKILL_TESTING_VERSION" >> $BASH_ENV
+      - run: mkdir -p workspace
+      - run: echo "export SKILL_TESTING_VERSION=$VERSION" >> workspace/new-env-vars
+      - run: echo $VERSION
+      - run: cd skill-testing-ml && git add package.json package-lock.json docs/*
+      - run:
+          name: commit changes
+          command: |
+            cd skill-testing-ml
+            git commit -m "Version $VERSION" -m "* $VERSION
+            * update docs"
+      - run: cd skill-testing-ml && git tag $VERSION
+      - run: cd skill-testing-ml && git tag -l
+      - run: cd skill-testing-ml && git push && git push --tags
+      - run: cd skill-testing-ml && npm publish --tag beta
+      - persist_to_workspace:
+          root: workspace
+          paths:
+              - new-env-vars
 
-
+  bst-client-release:
+    docker:
+      - image: circleci/node:8
+    working_directory: ~/bst-client-release
+    steps:
+      - add_ssh_keys:
+          fingerprints:
+            - "cf:77:b3:3a:08:72:b8:ad:bf:c4:11:97:3b:18:85:76"
+      - checkout
+      - attach_workspace:
+          at: ~/bst-client-release/workspace
+      - run: cat workspace/new-env-vars >> $BASH_ENV
+      - run: echo $SKILL_TESTING_VERSION
+      - run: rm -fr *
+      - run: git config --global user.email "$GITHUB_USER_EMAIL_RELEASE"
+      - run: git config --global user.name "$GITHUB_USER_NAME_RELEASE"
+      - run: echo "//registry.npmjs.org/:_authToken=$NPM_AUTH_TOKEN" > ~/.npmrc
+      - run: git clone git@github.com:bespoken/bst.git
```

Comment:
you shouldn't need the clone and install again, since the circle checkout already have access to the repository